lib/generators/rest_api_generator/templates/scope_rest_api_controller.rb.tt

```diff
@@ -0,0 +1,39 @@
+
+<%= version %>
```

Comment:
I think we can get this working with only one file with `module_namespacing` from rails:
https://github.com/rspec/rspec-rails/blob/main/lib/generators/rspec/controller/templates/controller_spec.rb
https://apidock.com/rails/v4.2.7/Rails/Generators/NamedBase/module_namespacing